src/transports.ts

```diff
@@ -13,6 +18,21 @@ export const fileTransport = async (logEntry: LogEntry) => {
   await fs.appendFile(logFilePath, JSON.stringify(logEntry) + "\n");
 };
 
-export const remoteLogsTransport = (logEntry: LogEntry) => {
-  // upload to remote logs storage service like sentry
+
+
+export const remoteLogsTransport = async (logEntry: LogEntry) => {
+  if (!logEntry.requestId) {
```

Comment:
Move these to a log function inside firebase.ts

Rename the firebase directory to transports and put other transports there, each in its own file with a log function.